components/schema-migrator/migrations/director/20230913091312_add_direction_to_apis_and_impl_standard_to_events.up.sql

```diff
@@ -0,0 +1,202 @@
+BEGIN;
+
+-- Drop views
+DROP VIEW IF EXISTS tenants_specifications;
+DROP VIEW IF EXISTS tenants_apis;
+DROP VIEW IF EXISTS tenants_events;
+
+CREATE TYPE api_direction_format AS ENUM (
+    'inbound',
+    'mixed',
+    'outbound'
+);
+
+-- Alter tables
+ALTER TABLE api_definitions
+    ADD COLUMN direction api_direction_format default 'inbound';
```

Comment:
Default value should be removed from here. 
According spec - if missing it should be assumed. We are not obligated to save it, but rather the consumer should assume it. 
`If not provided, "inbound" is assumed.`